### PR TITLE
fix: bug fix by adding descriptions in some packages

### LINKED ISSUE #1616: Descriptions in package.json
**Describe the bug**

some packages in monorepo don't have description, so npm and all other tools use readme 
for description that starts with svg image

**Screenshots**

Снимок экрана 2021-11-25 в 14 02 16

Снимок экрана 2021-11-25 в 14 01 07

**Expected behavior**
package.json have clear and concise description of what package does.

### PR DESCRIPTION
**Pull request by rjsduf0503**
Hello @jeetiss !

<b>Thanks for your preparing "good first issues" for us.</b> 
We found #1616 issue's bugs are appear in 8 packages and those were render, type, primitives, stylesheet, render, font, layout, and image.
<b>We added "clear and concise" descriptions of what package does as much as possible in package.json file in some repositories. </b>
So we could fix #1616 issue!
We wanted to add fixed image, but in our opinion, the BundlePhobia site seemed to be modified only when it was merged.

<b>If our fixes are wrong, please leave a comment.</b>

Thank you for reading this commit and your attention for us. Have a great day!!! 🙂

Desktop
- OS : Windows 10
- Browser : Chrome
- React-pdf version : 2.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,5 +83,7 @@
       "pre-commit": "git add . && lint-staged"
     }
   },
-  "dependencies": {}
+  "dependencies": {
+    "yarn": "^1.22.17"
+  }
```
Comment (jeetiss) [inline package.json:88]: please, remove `yarn` dependency and `package-lock.json` file

File: packages/font/package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "@react-pdf/font",
   "version": "2.0.14",
   "license": "MIT",
+  "description": "Register font and emoji source for PDF document",
```
Comment (jeetiss) [inline packages/font/package.json:5]:
```suggestion
  "description": "Register font and emoji source for react-pdf document",
```

File: packages/image/package.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "name": "@react-pdf/image",
   "version": "2.0.4",
   "license": "MIT",
+  "description": "Change the image url to png or jpeg format",
```
Comment (jeetiss) [inline packages/image/package.json:5]:
```suggestion
  "description": "Parses the images in png or jpeg format for react-pdf document",
```

File: packages/stylesheet/src/expand/borders.js [hunk 4]
```diff
@@ -11,7 +11,8 @@ const expandBorders = (key, value) => {
     const color = match[5] || value;
     const style = match[4] || value;
     const width = match[1] || value;
-
+    console.log(value);
+    console.log(match);
```
Comment (jeetiss) [inline packages/stylesheet/src/expand/borders.js:15]: remove, these too

File: yarn.lock [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 "@babel/cli@^7.2.3":
   version "7.13.14"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.13.14.tgz#c395bc89ec4760c91f2027fa8b26f8b2bf42238f"
+  resolved "https://registry.npmjs.org/@babel/cli/-/cli-7.13.14.tgz"
```
Comment (jeetiss) [inline yarn.lock:7]: revert the changes in `yarn.lock`, it for some reason change registry to npmjs in your PR

**Review by jeetiss — CHANGES_REQUESTED** [review]
Hi, folks! I added some comments

File: yarn.lock (hunk 5 above)
Comment (Happy0Hyun) [reply yarn.lock:7]: Thank you for leaving your reviews!! @jeetiss 
We'll send you the PR again after making the revisions to this as soon as possible.

And can we get another good first issue? We want to contribute more to this repository!
